packages/x-data-grid/src/components/columnHeaders/GridGenericColumnHeaderItem.tsx

```diff
@@ -111,7 +111,6 @@ const GridGenericColumnHeaderItem = React.forwardRef(function GridGenericColumnH
       tabIndex={tabIndex}
       aria-colindex={colIndex + 1}
       aria-sort={ariaSort}
-      aria-label={headerComponent == null ? label : undefined}
```

Comment:
I looked into this issue some time ago, but couldn't make it work properly.
Even with this change, it doesn't seem to work as expected – see the recording:


https://github.com/user-attachments/assets/14388fa9-5255-4a82-84b6-c52770f8bbf9

Any idea why it starts reading the column name twice?

Reply:
Which keys are you using to navigate the header items? 🤔

Reply:
`Control` + `Option` + arrow keys

Reply:
I have made another PR for that
https://github.com/mui/mui-x/pull/14484

Since it is out of the scope of the linked issue, I have split it to cover the bug faster

Reply:
I found the reason
it looks like the `Tooltip` component is adding another `aria-label` on the wrapped component

https://github.com/mui/mui-x/blob/cf441ece0f6c2e07acf49438ee14473bc4278b39/packages/x-data-grid/src/components/columnHeaders/GridColumnHeaderTitle.tsx#L77-L85

we don't pass it here, but consoling `other` in

https://github.com/mui/mui-x/blob/cf441ece0f6c2e07acf49438ee14473bc4278b39/packages/x-data-grid/src/components/columnHeaders/GridColumnHeaderTitle.tsx#L34-L50

shows `aria-label` prop

For string content, `aria-label` should not be attached to the element. For now, I will filter it out in our component, but I will make a PR to the component to fix it there as well

Reply:
> Control + Option + arrow keys

Ah makes sense, I tested with the arrow keys only, not the virtual cursor.

> I think that the voice over wants to connect header name with the cell when you use Control + Option + arrow keys. If you navigate through content you will notice that it reads out the data plus the column name which is great in that scenario.

Strange bug.

> Anyway, I think that we can ignore that issue and have at least regular arrow key navigation fixed

I tested again with the latest changes and it's working as expected with arrow keys at least 👍